src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -130,7 +130,11 @@ const HiearchicalGrid = ({
                       className="focusIndicatorDisplayBlock"
                     >
                       {isLive ? (
-                        <LiveLabel>{promo.title}</LiveLabel>
+                        <LiveLabel
+                          className={isFirstPromo ? 'first-promo' : undefined}
```

Comment:
We might be able to improve this using `&&` or `??` (I can never remember the difference between 😬 )

 ```suggestion
                          className={isFirstPromo ?? 'first-promo'}
```

Reply:
We tried this, and it complained, so we've decided to leave it for now

Reply:
Something like this could work, but not a big deal as is!
`<LiveLabel {...(isFirstPromo && { className: 'first-promo' })} />`

Reply:
This worked, thanks @amoore108 😄